### PR TITLE
spelling errors and inconsistent concurrency count in the example

### PR DESCRIPTION
**Pull request by jcavalieri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ JSON such as the following added as custom JSON to the stack:
       "slacker": {
         "process_count": 2,
         "config" : {
-          "concurrency": 5,
+          "concurrency": 10,
```
Comment (jcavalieri) [inline README.md]: The generated example YAML has 10, just making it consistent.

**Comment by alanpeabody** [comment]
I am not sure anyone is maintaining this anymore. :-/

**Comment by jcavalieri** [comment]
Oh. @alanpeabody, do you know if there is there a fork that is maintained?

**Comment by alanpeabody** [comment]
@jcavalieri I am not, but there do seem to be some good PRs open that might be worth getting merged together in a fork.

**Pull request closed without merging**